controller/internal/routes/current_identity_router.go

```diff
@@ -184,6 +203,8 @@ func (r *CurrentIdentityRouter) removeMfa(ae *env.AppEnv, rc *response.RequestCo
 		return
 	}
 
+	ae.Handlers.PostureResponse.SetMfaPostureForIdentity(rc.Identity.Id, false)
```

Comment:
On removing MFA from an identity this will set the posture data for mfa to "not passing" for all API Sessions.